website/docs/r/s3_bucket_analysis_configuration.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
Documentation page is missing sidebar link in `website/aws.erb` (required until documentation is moved off terraform.io in the coming months)

Reply:
🤦‍♂ Yeah, I've asked for that change a few times…